### PR TITLE
Infinite canvas now scales on any screen resolution

### PR DESCRIPTION
**Pull request by irishgreencitrus**
Closes issue #1

### CODE REVIEW

File: lorien/InfiniteCanvas/InfiniteCanvas.gd [hunk 1]
```diff
@@ -34,6 +35,7 @@ func _ready():
 	_active_tool._on_brush_color_changed(_brush_color)
 	_active_tool._on_brush_size_changed(_brush_size)
 	_active_tool.enabled = true
+	get_tree().get_root().connect("size_changed",self,"_on_window_resized")
```
Comment (mbrlabs) [inline lorien/InfiniteCanvas/InfiniteCanvas.gd:38]: Just a style nitpick: can you add spaces between function arguments?

File: lorien/InfiniteCanvas/InfiniteCanvas.gd [hunk 2]
```diff
@@ -280,3 +282,6 @@ func clear() -> void:
 	info.point_count = 0
 	info.stroke_count = 0
 	_current_project.strokes.clear()
+# -------------------------------------------------------------------------------------------------
```
Comment (mbrlabs) [inline lorien/InfiniteCanvas/InfiniteCanvas.gd:285]: Please add a newline beween functions

**Review by mbrlabs — CHANGES_REQUESTED** [review]
Nice, that was quick 👍 
I just left some code formatting comments, otherwise looks good!

**Comment by irishgreencitrus** [comment]
Sure thing! I knew you had a code style i just missed a bit of it

**Comment by irishgreencitrus** [comment]
There you go @mbrlabs

**Comment by irishgreencitrus** [comment]
There you go @mbrlabs

**Comment by mbrlabs** [comment]
Thanks!

**Pull request merged**